hs_core/models.py

```diff
@@ -610,10 +610,13 @@ def update(cls, element_id, **kwargs):
                 except TypeError:
                     raise TypeError("Not a valid date value.")
                 if dt.type == 'created':
-                    raise ValidationError("Resource creation date can't be changed")
+                    #raise ValidationError("Resource creation date can't be changed")
+                    dt.start_date = start_dt
+                    dt.save(force_update=True)
                 elif dt.type == 'modified':
-                    dt.start_date = now().isoformat()
-                    dt.save()
+                    # dt.start_date = now().isoformat()
+                    dt.start_date = start_dt
+                    dt.save(force_update=True)
```

Comment:
The template is using the resource.updated field.

https://github.com/hydroshare/hydroshare/blob/develop/hs_core/templates/pages/genericresource.html#L125